phpunit.evergreen.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.3/phpunit.xsd"
+        backupGlobals="false"
+        backupStaticAttributes="false"
+        colors="true"
+        convertErrorsToExceptions="true"
+        convertNoticesToExceptions="true"
+        convertWarningsToExceptions="true"
+        stopOnFailure="false"
+        syntaxCheck="false"
+        bootstrap="tests/bootstrap.php"
+>
+
+    <php>
+        <ini name="error_reporting" value="-1"/>
+        <env name="MONGODB_URI" value="mongodb://127.0.0.1:27017/?serverSelectionTimeoutMS=100"/>
```

Comment:
Should this be omitted so we pick up the environment variable from `run_tests.sh`, or is this only a fallback?

Reply:
This is a default so that we don't pass an empty uri if not specified externally. From the [PHPUnit manual on `php.env`](https://phpunit.readthedocs.io/en/9.5/configuration.html#the-env-element):
> By default, environment variables are not overwritten if they exist already. To force overwriting existing variables, use the `force` attribute.